components/definitions/chromel.d.ts

```diff
@@ -201,9 +201,11 @@ declare namespace chrome.braveShields {
 
   type BraveShieldsViewPreferences = {
     showAdvancedView: boolean
+    statsBadgeVisible: boolean
   }
   type BraveShieldsSetViewPreferencesData = {
     showAdvancedView?: boolean
+    statsBadgeVisible?: boolean
   }
```

Comment:
I think both these types can be removed since we use `chrome.settingsPrivate` API now. Must have left this in by accident.